src/ConnectionGraphicsObject.cpp

```diff
@@ -106,7 +106,7 @@ nodeScene() const
 }
 
 
-ConnectionId
+const ConnectionId &
```

Comment:
`ConnectionId const &`
https://mariusbancila.ro/blog/2018/11/23/join-the-east-const-revolution/

Reply:
oh, you are one of those ;)